requirements.txt

```diff
@@ -19,3 +19,4 @@ user-agents>=1.1.0
 filelock>=3.0.1
 setuptools
 jellyfish>=0.6.1
+pyyaml
```

Comment:
It's an optional dependency right now, what is reason for promoting it here?

Reply:
Because the project I use for testing is Rails and I needed it all to get running quickly. Will be removed in final version since I now hear from you it's optinal.